library.gradle

```diff
@@ -43,20 +43,6 @@ android {
     }
 }
 
-dependencies {
-    testImplementation 'androidx.test:core:1.4.0'
-    testImplementation 'androidx.test:runner:1.4.0'
-    testImplementation 'androidx.test:rules:1.4.0'
-    testImplementation 'androidx.test.ext:junit:1.1.3'
-    testImplementation 'org.robolectric:robolectric:4.6.1'
```

Comment:
Note that the versions on the `library.gradle` file were higher in general than the ones on each module. I believe when there is a conflict of versions, gradle will pick the one with the highest version. So I used the higher version libraries for all the modules. These are all test dependencies so it shouldn't affect users in any case